File: Basic/Java/Problem_12.java
import java.util.*;  
class Swap_With {  
    public static void main(String[] args) {  
       int x, y, t;
       Scanner sc = new Scanner(System.in);  
       System.out.println("Enter the value of X and Y");  
       x = sc.nextInt();  
       y = sc.nextInt();  
       System.out.println("before swapping numbers: "+x +"  "+ y);   
       t = x;  
       x = y;  
       y = t;  
       System.out.println("After swapping: "+x +"   " + y);  
       System.out.println( );  
    }    
} 

File: Basic/C#/problem_12.c
/*
Input two variables and swap them using a function.
*/

// Solved by Bhumi54321

#include <stdio.h>

void swap(int *p, int *q){
    *p = *p + *q - (*q = *p);
}
int main(){
    int a,b;
    printf("Enter the value of a and b: ");
    scanf("%d %d",&a,&b);
    swap(&a,&b);
    printf("The values of a and b after swapping:\na = %d\nb = %d",a,b);
    return 0;
}
